pkgs/by-name/pl/plus-jakarta-sans/package.nix

```
@@ -0,0 +1,32 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchzip,
+}:
+stdenvNoCC.mkDerivation rec {
```

```suggestion
stdenvNoCC.mkDerivation (finalAttrs: {
```
more desired than rec, wherever possible